### PR TITLE
fix(core): adapt gorgone's CI integration

### PR DESCRIPTION
**Pull request by sc979**
Add new global mandatory configuration file : config.yaml
Modify file path in the service configuration
Package and install the config.yaml file
Partially update github's documentation

image

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -134,7 +134,8 @@ $ systemctl status centreon-gorgone
    Active: active (running) since Mon 2019-09-30 09:36:19 CEST; 2min 29s ago
  Main PID: 5168 (perl)
    CGroup: /system.slice/centreon-gorgone.service
-           ├─5168 /usr/bin/perl /usr/bin/gorgoned --config=/etc/centreon/gorgoned.yml --logfile=/var/log/centreon/gorgoned.log --severity=error
+           ├─5168 /usr/bin/perl /usr/bin/gorgoned --config=/etc/centreon-gorgone/config.yaml
```
Comment (sc979) [inline docs/getting_started.md]:
```suggestion
           ├─5168 /usr/bin/perl /usr/bin/gorgoned --config=/etc/centreon-gorgone/config.yaml --logfile=/var/log/centreon/gorgoned.log --severity=error

```

File: docs/getting_started.md [hunk 2]
```diff
@@ -134,7 +134,9 @@ $ systemctl status centreon-gorgone
    Active: active (running) since Mon 2019-09-30 09:36:19 CEST; 2min 29s ago
  Main PID: 5168 (perl)
    CGroup: /system.slice/centreon-gorgone.service
-           ├─5168 /usr/bin/perl /usr/bin/gorgoned --config=/etc/centreon/gorgoned.yml --logfile=/var/log/centreon/gorgoned.log --severity=error
+           ├─5168 /usr/bin/perl /usr/bin/gorgoned --config=/etc/centreon-gorgone/config.yaml --logfile=/var/log/centreon/gorgoned.log --severity=error
+
```
Comment (sc979) [inline docs/getting_started.md]:
```suggestion
```

File: docs/getting_started.md [hunk 3]
```diff
@@ -134,7 +134,8 @@ $ systemctl status centreon-gorgone
    Active: active (running) since Mon 2019-09-30 09:36:19 CEST; 2min 29s ago
  Main PID: 5168 (perl)
    CGroup: /system.slice/centreon-gorgone.service
-           ├─5168 /usr/bin/perl /usr/bin/gorgoned --config=/etc/centreon/gorgoned.yml --logfile=/var/log/centreon/gorgoned.log --severity=error
+           ├─5168 /usr/bin/perl /usr/bin/gorgoned --config=/etc/centreon-gorgone/config.yaml --logfile=/var/log/centreon/gorgoned.log --severity=error
+           --logfile=/var/log/centreon/gorgoned.log --severity=error
```
Comment (sc979) [inline docs/getting_started.md]:
```suggestion
```

File: docs/getting_started.md [hunk 4]
```diff
@@ -151,7 +151,7 @@ Sep 30 09:36:19 cga-centreon-19-10.int.centreon.com systemd[1]: Started Centreon
 If you are using the sources, execute the following command:
 
 ```bash
-perl gorgoned --config=config/gorgoned.yml --severity=error
+perl gorgoned --config=config/config.yaml --severity=error
```
Comment (sc979) [inline docs/getting_started.md]: @garnier-quentin , please check the consistency of this line

Comment (garnier-quentin) [reply docs/getting_started.md]: gorgoned can use that file (if the file is well formatted)

Comment (sc979) [reply docs/getting_started.md]: isn't more like : config=/etc/centreon-gorgone/config.yaml ? like on line 137

Comment (garnier-quentin) [reply docs/getting_started.md]: it's for source

**Pull request merged**